source/typescript.txt

```diff
@@ -0,0 +1,35 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since Typescript builds on JavaScript, you can use the
+`Node.js Driver <https://www.mongodb.com/docs/drivers/node/current/>`__
+to create TypeScript applications that communicate with MongoDB.
+
+The Node.js Driver started supporting TypeScript with version 4.0. You must
```

Comment:
S: "started supporting" makes it sound like it hasn't finished supporting TypeScript (which, from a technical perspective, is probably accurate, because when is support ever _totally_ complete, but it makes it sound like we have a half-baked driver).

```suggestion
The Node.js Driver began support for TypeScript with version 4.0. You must
```

Or:

```suggestion
Support for TypeScript began in version 4.0 of the Node.js Driver. You must
```

Or:

```suggestion
Versions 4.0 and above of the Node.js Driver support TypeScript. You must
```